components/NavigationBar/tests/unit/NavigationBarTests.m

```diff
@@ -136,4 +138,128 @@ - (void)testEncoding {
   XCTAssertEqual(navBar.titleAlignment, unarchivedBar.titleAlignment);
 }
 
+
+#pragma mark - Accessibility
+
+- (void)testNavigationBarIsNotAccessibilityElement {
+  // Given
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+
+  // Then
+  XCTAssertFalse(navBar.isAccessibilityElement);
+}
+
+- (void)testAccessibilityItemsCountWithNoTitle {
+  // Given
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+
+  // Then
+  XCTAssertEqual(3, navBar.accessibilityElementCount);
```

Comment:
How are we choosing this number? What would it mean if this test were to fail in the future?

Reply:
May be helpful to document the number here.

Reply:
Canceled travis

Reply:
Done.